### ISSUE TITLE
The Islands

### ISSUE THREAD

**Issue by 00SalAnd** [Monika-After-Story/MonikaModDev#3034]
I'm sorry if I sound dumb but I hear a lot of people talking about the islands and I have been wondering if it is something that needs to be unlocked or if I just have to wait until it is brought up by Monika naturally. Thanks in advance.

**Comment by Rai99** [comment]
The islands unlock at 400 affection.

**Comment by AsuraShun** [comment]
speaking of which,is the building still going to be completed in another update? or is it something that can be accessed now?

**Comment by AsuraShun** [comment]
i'm already at love+ and the fact that it isn't finish made me curious

**Comment by MaximumRustler** [comment]
Yeah, I'm pretty sure that unfinished part is for a later update. I'm also 1000+ affection and haven't seen that bit change.

**Comment by 00SalAnd** [comment]
Ok thanks a lot

**Comment by JellicleDry** [comment]
Wait a minute, how do you find the exact affection level that Monika is at with you?

**Comment by AsuraShun** [comment]
the affection level is in the aff log is in the log folder
